public/components/notebooks/components/note_table.tsx

```diff
@@ -80,6 +98,8 @@ export function NoteTable({
   const [searchQuery, setSearchQuery] = useState('');
   const location = useLocation();
   const history = useHistory();
+  const [_dataSourceMDSId, setDataSourceMDSId] = useState<string | undefined>('');
+  const [_dataSourceMDSLabel, setDataSourceMDSLabel] = useState<string | undefined>('');
```

Comment:
unused?

Reply:
this I dont use the variable but I have used the set function